### ISSUE TITLE
After change username/password the app doesn't request the credentials

### ISSUE THREAD

**Issue by abreudiego** [TwidereProject/Twidere-Android#840]
Check before submitting issue -- Thank you!

You can ignore this form if this issue is a feature request.

## Expected behavior
When the password or username is changed the app should request the credentials again.
## Actual behavior
The app doesn't request the credentials and keeps showing the old username
## Steps to reproduce
Change the username or password by twitter website and uses the app after that.
## Extra info

Android version: (version code here)
5.1.1 LMY47V
Network type:

- [X] Cellular
- [X] Wi-Fi

App version: (version code here)
3.6.11
Build variant:

- [ ] Google
- [ ] F-Droid

Micro-blogging service:

- [X] Twitter
- [ ] Fanfou
- [ ] GNU Social / StatusNet
- [ ] Mastodon

App settings:

- [ ] Custom API
- [ ] Custom consumer key/secrets
- [ ] Official keys
- [ ] Using [Thumbor](https://github.com/thumbor/thumbor)
- [ ] Connected through proxy
- [ ] Media preview enabled

**Comment by mariotaku** [comment]
Yes, this is expected and has nothing to do with Twidere. To understand
why, we'll need to know how OAuth work.

TL;DR: Password and app permission are different.

Full explanation: When you sign in to Twitter using Twidere, it doesn't
store your password actually. Instead, it obtains a token with permission
to some of your Twitter data, for accessing Twitter via API. If you need to
revoke Twidere's access, you can go to https://twitter.com/settings/apps .
Diego de Abreu <notifications@github.com>于2017年5月18日 周四08:36写道：
